packages/react_rearch/README.md

```diff
@@ -0,0 +1,7 @@
+# React ReAarch
+
+##  1.  Introduction
+
+ReArch port to React.
+The code structure is copied and adapted from [flutter_rearch](https://pub.dev/packages/flutter_rearch).
+
```

Comment:
Would you mind just changing this entire file to:

```md
# React ReArch

Experimental, and not fully supported, port of ReArch to React in Dart.
```

And then maybe throw a quick example below it.